packages/contentstack-export/src/lib/export/locales.js

```diff
@@ -12,19 +12,14 @@ const helper = require('../util/helper');
 const { addlogs } = require('../util/log');
 let config = require('../../config/default');
 let localeConfig = config.modules.locales;
-const masterLocale = config.master_locale;
 let requiredKeys = localeConfig.requiredKeys;
 let stack = require('../util/contentstack-management-sdk');
 let client
 function ExportLocales() {
   this.qs = {
     include_count: true,
     asc: 'updated_at',
-    query: {
-      code: {
-        $nin: [masterLocale.code],
-      },
-    },
+    query: {},
```

Comment:
Why removed the condition to not fetch the master local?
We do not export the master local for any stack.
Please discuss with @shafeeqd959 for this change.

Reply:
@netrajpatel This condition is handled across import/export script. We can connect and walk through the code.
We need to store uid of master locale from the source stack in order to properly migrate custom roles and locales.
cc: @shafeeqd959

Reply:
@vkalta It will be good if we make changes of adding/deleting locals in the 'custom roles' lib file rather than changing the core logic it has lot of other dependancies.
CC: @shafeeqd959